### PR TITLE
Fix description HISB

### LINKED ISSUE #1847: Fix description and spelling...
We need to fix the description for the hisb module in the download.

It should say Strongs (with a 2nd "s"), capitalize the word English, and spell transliteration correctly:
Screenshot_20211013-085804_Bible Study

### PR DESCRIPTION
**Pull request by timbze**
Fixes https://github.com/AndBible/and-bible/issues/1847

### CODE REVIEW

**Comment by tuomas2** [comment]
I recently created repo for these special modules, so could you also update here:

https://github.com/AndBible/special-modules/tree/master/HISB

File: andbible/mods.d/HISB.conf [hunk 1]
```diff
@@ -4,7 +4,7 @@ Lang=he
 Encoding=UTF-8
 Direction=RtoL
 Version=1.3
```
Comment (tuomas2) [inline andbible/mods.d/HISB.conf:6]: Version number should be incremented
```suggestion
Version=1.4
```

**Comment by tuomas2** [comment]
> I recently created repo for these special modules, so could you also update here:
> 
> https://github.com/AndBible/special-modules/tree/master/HISB

I made it now in https://github.com/AndBible/special-modules/commit/63dcf4f72bee984dc85cfdb6b0a10c945c21e9bf

File: andbible/mods.d/HISB.conf [hunk 2]
```diff
@@ -3,8 +3,8 @@ DataPath=./modules/texts/ztext/HISB/
 Lang=he
 Encoding=UTF-8
 Direction=RtoL
-Version=1.3
-Description=Hebrew with Strong and english translitration. Works properly with And Bible 4.0+
+Version=1.4
```
Comment (tuomas2) [inline andbible/mods.d/HISB.conf]:
```suggestion
Version=1.3
```

maybe version update not needed because of this ... after all.

**Comment by tuomas2** [comment]
BTW conf file inside zip is not updated, so it will show up correctly in downloads screen but not in document selector.

**Review by tuomas2 — CHANGES_REQUESTED** [review]
Update zip file

**Comment by timbze** [comment]
Updated zip now

**Comment by tuomas2** [comment]
Thanks!

**Pull request merged**